doc/how_to/custom_components/reactive_html/reactive_html_panes.md

```diff
@@ -0,0 +1,150 @@
+""",
+    }
+    __javascript__ = [
+        "https://cdn.jsdelivr.net/npm/chart.js@4.4.0/dist/chart.umd.min.js"
+    ]
+
+
+def data(chart_type="line"):
+    return {
+        "type": chart_type,
+        "data": {
+            "labels": ["January", "February", "March", "April", "May", "June", "July"],
+            "datasets": [
+                {
+                    "label": "Data",
+                    "backgroundColor": "rgb(255, 99, 132)",
+                    "borderColor": "rgb(255, 99, 132)",
+                    "data": [0, 10, 5, 2, 20, 30, 45],
+                }
+            ],
+        },
+        "options": {
+            "responsive": True,
+            "maintainAspectRatio": False,
+        },
+    }
+
+
+chart_type = pn.widgets.RadioBoxGroup(
+    name="Chart Type", options=["bar", "line"], inline=True
+)
+grid = ChatJSComponent(
+    object=pn.bind(data, chart_type), height=400, sizing_mode="stretch_width"
+)
+pn.Column(chart_type, grid).servable()
+```
+
+Initially I tried creating the chart in the `render` function, but the chart did not display.
+I found out via Google Search and experimentation that the chart needs to be created later in the
+`after_layout` function.
+
+## Creating a Cytoscape Pane
+
+```{pyodide}
+import param
+import panel as pn
+from panel.reactive import ReactiveHTML
+
+class Cytoscape(ReactiveHTML):
+    object = param.List()
+
+    layout = param.Selector(default="cose", objects=["breadthfirst", "circle", "concentric", "cose", "grid", "preset", "random"])
+    style = param.String("", doc="Use to set the styles of the nodes/edges")
+
+    zoom = param.Number(1, bounds=(0,100))
+    pan = param.Dict({"x": 0, "y": 0})
+
+    data = param.List(doc="Use to send node's data/attributes to Cytoscape")
+
+    selected_nodes = param.List()
+    selected_edges = param.List()
+
+    _template = """
+        <div id="cy" style="width: 100%; height: 100%; position: relative; border: 1px solid"></div>
+    """
+    __javascript__ = ['https://cdnjs.cloudflare.com/ajax/libs/cytoscape/3.23.0/cytoscape.umd.js']
+
+    _scripts = {
+        'render': """
+            self.create()
+        """,
+        "create": """
+            if (state.cy == undefined){
+                state.cy = cytoscape({
+                container: cy,
+                layout: {name: data.layout},
+                elements: data.object,
+                zoom: data.zoom,
+                pan: data.pan,
+                });
+                state.cy.on('select unselect', function (evt) {
+                  data.selected_nodes = state.cy.elements('node:selected').map((el)=>{return el.id()})
+                  data.selected_edges = state.cy.elements('edge:selected').map((el)=>{return el.id()})
+                });
+                self.style()
+                const mainEle = document.querySelector("body")
+                mainEle.addEventListener("scrollend", (event) => {state.cy.resize().fit()})
```

Comment:
Don't we lack a life cycle function after `after_layout` @philippjfr ? Why is a hack like listening for `scrollend` or adding a `timeOut` sometimes needed?

Reply:
In https://github.com/holoviz/panel/issues/5195#issuecomment-1693935047 @derrikmcs also notes that `scrollend` event is not supported by all browsers.

Reply:
There sadly is no event after `after_layout` as that's the last step in the Bokeh rendering pipeline.